pkg/cli/subcommands/autoscalerlist/command.go

```diff
@@ -0,0 +1,128 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2025-present Datadog, Inc.
+
+// Package autoscalerlist implements 'agent autoscaler-list'.
+package autoscalerlist
```

Comment:
Do you think it makes sense to add this folder to CODEOWNERS so that your team owns it? CLI commands default to agent-configuration's ownership, but specific commands can be owned by teams, for example `/pkg/cli/subcommands/clusterchecks` is owned by `container-platform`.

Reply:
I added this line here which should update the ownership going forward (but let me know if it looks misconfigured!): https://github.com/DataDog/datadog-agent/pull/36417/files#diff-3d36a1bf06148bc6ba1ce2ed3d19de32ea708d955fed212c0d27c536f0bd4da7R371

Reply:
LGTM!